taker-electron/src/main/main.ts

```diff
@@ -10,13 +10,15 @@
  */
 import { app, BrowserWindow, net, shell } from "electron";
 import log from "electron-log";
+import nodenet from "node:net";
 import path from "path";
 import MenuBuilder from "./menu";
 import { resolveHtmlPath } from "./util";
 // eslint-disable-next-line import/no-unresolved
 const { itchysats } = require("../../index.node");
 
 let mainWindow: BrowserWindow | null = null;
+let port = 8000;
```

Comment:
I think we can remove this assignment as it's confusing? (it changes before first use)
```suggestion
let port = 8000;
```